planning/scenario_planning/parking/astar_search/src/astar_search.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "astar_search/astar_search.hpp"
+
```

Comment:
Curious, why was a new line added here?

Reply:
As the "corresponding header file", I thought it should be in its own group. Cpplint thinks so too: If there were any C++ stdlib headers, they would be put after this header, but before the helper header by cpplint.

Reply:
i see, thanks for the explanation!